charts/gardener-extension-provider-openstack/templates/deployment.yaml

```diff
@@ -53,6 +53,9 @@ spec:
         - --webhook-config-server-port={{ .Values.webhookConfig.serverPort }}
         - --disable-controllers={{ .Values.disableControllers | join "," }}
         - --disable-webhooks={{ .Values.disableWebhooks | join "," }}
+        {{- if .Values.healthPort }}
```

Comment:
Will `.Values.healthPort` not always set? There is a default value in the `values.yaml` defined.

Reply:
Since `health-bind-address` has a default value, it can be skipped from here.

Reply:
I guess you mean the default value you set in the `values.yaml` or the one which you define in the code? As far as I see they are both the same. My point that the if clause wouldn't be required here, when the chart has a default value, right? Same for the admission controller. But it is just a nit.

Reply:
My idea was if someone uses the same yamls but don't give the value of `healthPort` in values.yaml in that case it will throw an error, though not the case in our scenario. What's you call? Should I remove the `if`?

Reply:
The `.Values.healthPort` will not be empty as you define a value for it in the `values.yaml` of the chart. This can be overridden in case someone want so provide own value for it, but your default value will be there.  As said it does not hurt to have the `if` but it's not required in my opinion. 
So up to you I'm fine with keeping or removing it as it is just a nit.

Reply:
Remove it in c2a041cf94eed49c66d41676f5a911163e6b59bf. PTAL